packages/playwright/src/index.ts

```diff
@@ -22,7 +22,9 @@ import { setBoxedStackPrefixes, asLocator, createGuid, currentZone, debugMode, i
 
 import { currentTestInfo } from './common/globals';
 import { rootTestType } from './common/testType';
+import { fixTestPrompt } from './reporters/base';
```

Comment:
Let's not reach into reporters from the test harness implementation, that would be a layering violation. Extract it into a separate file or inline into index.ts. We should fix DEPS to yell on these.

Reply:
done